src/Core/src/AwsClientFactory.php

```diff
@@ -145,6 +146,19 @@ public function dynamoDb(): DynamoDbClient
         return $this->serviceCache[__METHOD__];
     }
 
+    public function ecr(): EcrClient
+    {
+        if (!class_exists(EcrClient::class)) {
+            throw MissingDependency::create('async-aws/ecr', 'DynamoDb');
```

Comment:
```suggestion
            throw MissingDependency::create('async-aws/ecr', 'ECR');
```

Reply:
ha, it's wrong because i copied IAM, which is also wrong

Reply:
Oh..